bundled/boost-1.62.0/include/boost/config/compiler/visualc.hpp

```diff
@@ -294,9 +294,10 @@
 #endif
 
 //
-// tjhei: taken from boost git repo
+// tjhei: upgrade supported MSVC version to 19.11
+// Boost repo has only 19.10:
 // last known and checked version is 19.10.24629 (VC++ 2017 RC):
-#if (_MSC_VER > 1910)
+#if (_MSC_VER > 1911)
```

Comment:
What a sad state of affairs that the BOOST people have so little trust in Microsoft that they explicitly blacklist all future compilers they have not yet tried...